### ISSUE TITLE
Improve Hair Hurt Animations

### ISSUE THREAD

**Issue by jrconway3** [makrohn/Universal-LPC-spritesheet#45]
The frames for the Hair Hurt animations are rather off. I've been noticing this for a while, but the hair animations should probably be updated to show the hair further down on the head and actually show the hair as being on the top of the head.

I was just copy-pasting the base hairstyles, but later noticed that they didn't look right. After noticing the the hood animations DO, in fact, turn as the head turns down, I realized that we definitely should animate the hurt frames better. It probably only needs to be about two frames, though.

This isn't a high priority, but I will work on it later.

**Comment by makrohn** [comment]
Agreed that this is a problem - I was mostly just trying to make dying/hurt characters not go magically bald.
